### DISCUSSION TITLE
Bem-vindo às discussões da JethJS!

### DISCUSSION THREAD

**Discussion by solundev** [RootGC/JethJS#2] [General]
## 👋 Bem-vindo!
  Estamos usando as Discussões como um lugar para nos conectarmos com outros membros da nossa comunidade. Esperamos que você:
  * Faça perguntas sobre as quais você está se perguntando.
  * Compartilhar ideias.
  * Envolva-se com outros membros da comunidade.
  * Dê boas-vindas aos outros e tenha a mente aberta. Lembre-se que esta é uma comunidade que
  construir juntos 💪.

  Para começar, comente abaixo com uma apresentação sua e conte-nos sobre o que você faz com esta comunidade.

**Comment by Nikii-Discord** [comment]
Hello Soso!! Como se está?

**Reply by Nikii-Discord** [reply]
Senti sdds e fui ver como estão as coisas 🙄

**Reply by solundev** [reply]
👍
